src/main/car/inventory/DealerGroup.java

```diff
@@ -2,19 +2,28 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 
 public class DealerGroup {
 
 	private ArrayList<Dealership> dealers;
-	
+
 	public DealerGroup() {
 		this.dealers = new ArrayList<>();
 	}
 
-	public ArrayList<Dealership> getDealers() {
```

Comment:
please change this back to getDealers, it does not get a DealerGroup, but rather the dealers